components/patient-access-rules/ui/src/main/resources/PhenoTips/PatientAccessRightsManagement.xml

```diff
@@ -1065,7 +1065,11 @@ $xwiki.ssx.use($serviceDocName)
   #elseif ($ownerReference.equals($xcontext.userReference))
     #set ($ownerName = 'you')
   #else
-    #set ($ownerName = $xwiki.getUserName("$ownerReference", false))
+    #if($owner.isGroup())
+      #set ($ownerName = $xwiki.getDocument($ownerReference.toString()).getTitle())
```

Comment:
There's no need to call `ownerReference.toString()`, for several reasons:

- Velocity automatically converts parameters to strings, if a method with the requested name and provided parameter type doesn't exist but a method with `String` parameters exists
- using strings as document names is deprecated, you should try to use `DocumentReference`s as often as possible
- while `Owner.getUser()` officially returns an `EntityReference`, the actual type is `DocumentReference`, and there is an `XWiki.getDocument(DocumentReference)` method, which is the best one to use